updated color discussion (#58)
MrLongbottom/P10-Article: sections/Discussion.tex

The article is about agriculture and how farmers are letting their cows out onto grass fields in September. 
It also mentions a few different farms in the Northern part of Jutland and describes these in various ways.

> \begin{table}
> 	\begin{tabular}{c|p{0.8\columnwidth}}
> 		Topic & Top 10 words for \gls{lda} \\
> 		\toprule
> 		0 & 'sæby', 'a', 'direktør', 'frederikshavn', 'virksomheden', 'hans', 'medarbejdere', 'pedersen', 'firmaet', 'procent' \\
> 		1 & 'omradet', 'boliger', 'natur', 'naturen', 'ligger', 'du', 'vand', 'dyr', 'a', 'skov' \\
> 		2 & 'nielsen', 'arets', 'prisen', 'dansk', 'mors', 'jensen', 'løgstør', 'aars', 'vm', 'thy' \\
> 	\end{tabular}
> 	\begin{tabular}{c|p{0.8\columnwidth}}
> 		Topic & Top 10 words for Author-Topic \\
> 		\toprule
> 		0 & 'du', 'procent', 'unge', 'børn', 'arige', 'hans', 'dansk', 'mig', 'thisted', 'mener'\\
> 		1 & 'sine', 'skriver', 'mig', 'børn', 'seneste', 'land', 'dansk', 'kommuner', 'andersen', 'formand' \\
> 		2 & 'set', 'glas', 'odense', 'vesthimmerland', 'leth', 'markedet', 'trump', 'ni', 'regionerne', 'prins'\\
> 	\end{tabular}
> 	\begin{tabular}{c|p{0.8\columnwidth}}
> 		Topic & Top 10 words for Category-Topic \\
> 		\toprule
> 		0 & 'du', 'hans', 'børn', 'mig', 'thisted', 'procent', 'bedre', 'a', 'kr', 'maske' \\
> 		1 & 'klar', 'haft', 'fem', 'hjørring', 'ham', 'nyt', 'formand', 'min', 'aften', 'sagen'\\
> 		2 & 'du', 'min', 'gode', 'gamle', 'ad', 'henrik', 'eu', 'finde', 'sat', 'hobro'\\
> 	\end{tabular}
> 	\caption{Top 10 words for each model used in \autoref{tab:disc_color}}
> 	\label{tab:top_words_three_models}
> \end{table}

Reviewer: ```suggestion
\begin{table}
	\caption{Top 10 words for each model used in \autoref{tab:disc_color}.}
	\label{tab:top_words_three_models}
	\begin{tabular}{c|p{0.8\columnwidth}}
		Topic & Top 10 words for \gls{lda} \\
		\midrule
		0 & sæby, a, direktør, frederikshavn, virksomheden, hans, medarbejdere, pedersen, firmaet, procent \\
		1 & omradet, boliger, natur, naturen, ligger, du, vand, dyr, a, skov \\
		2 & nielsen, arets, prisen, dansk, mors, jensen, løgstør, aars, vm, thy \\
	\end{tabular}
	\begin{tabular}{c|p{0.8\columnwidth}}
		\midrule
		Topic & Top 10 words for Author-Topic \\
		\midrule
		0 & du, procent, unge, børn, arige, hans, dansk, mig, thisted, mener\\
		1 & sine, skriver, mig, børn, seneste, land, dansk, kommuner, andersen, formand \\
		2 & set, glas, odense, vesthimmerland, leth, markedet, trump, ni, regionerne, prins\\
	\end{tabular}
	\begin{tabular}{c|p{0.8\columnwidth}}
		\midrule
		Topic & Top 10 words for Category-Topic \\
		\midrule
		0 & du, hans, børn, mig, thisted, procent, bedre, a, kr, maske \\
		1 & klar, haft, fem, hjørring, ham, nyt, formand, min, aften, sagen\\
		2 & du, min, gode, gamle, ad, henrik, eu, finde, sat, hobro\\
	\end{tabular}
\end{table}
```
Tydeligere separering mellem modeller, og har fjernet apostroffer så der er færre newlines.

> In \autoref{tab:top_words_three_models}, we can see that there is not many top words in the Author-topic and Category-Topic model that appear in the article below.

Reviewer: ```suggestion
In \autoref{tab:top_words_three_models}, we can see that there are not many top words in the Author-Topic and Category-Topic model that appear in the article below.
```

> In order to compare these models, we take the top 200 words for each topic-word distribution within each model and marking them in the article below.

Reviewer: ```suggestion
To compare these models, we take the top 200 words for the topic-word distributions within each model and mark them in the article below.
```

> We have taken 200 words, since we want to see how intertwined the models, but it is hard to find specific words since the topic coherence is low for Author and Category-Topic model.

Reviewer: ```suggestion
We take 200 words since we want to see how intertwined the models are and because it is hard to find specific words for the Author and Category-Topic model since the topic coherence is low.
```

Author-topic specific words are not very present and are only showing three unique words: "byder", "hører", and "børnene".
This indicates that the Author-topic model has trouble generalizing what the author of this article (Peter Tordrup Larsen) is writing about, possibly because he has written $5002$ articles in our dataset.
Another aspect of the Author-Topic model is that the authors writing these articles most likely do not write about just one subject, which explains why there is only three less important words marked here. 

> The Category-topic model only shows four unique words: "søndag"(sunday), "leg"(play), "klokken"(clock), and "danskere"(danish people).

Reviewer: ```suggestion
The Category-topic model only shows four unique words: "søndag" (sunday), "leg" (play), "klokken" (clock), and "danskere" (Danes).
```

These words are also very abstract and can be used in many different scenarios.

An interesting part of this analysis is the words appearing in all models.

> Some of the words within this category are: "dansk"(danish), "millioner"(millions), "samt"(including), and "nordjyske"(north jutland).

Reviewer: ```suggestion
Some of the words within this category are: "dansk" (Danish), "millioner" (millions), "samt" (including), and "nordjyske" (North Jutland).
```

> These words are not very descriptive, about what the article, but it is also hard to summarize this text since it covers a wide variety of specific topics.

Reviewer: ```suggestion
These words are not very representative of the content of the article, but it is also hard to summarize this text since it covers a wide variety of specific topics.
```

> Combining the results of these models might yield better topic models, since some words appearing in two models are "medarbejder"(coworker), "arbejdet"(the work), and "dyr"(animals).

Reviewer: ```suggestion
Combining the results of these models might yield better topic models since some words appearing in two models are "medarbejder" (coworker), "arbejdet" (the work), and "dyr" (animals).
```

> However, there are also words, such as "otto"(eight), "september", and "ligesom"(like), which are not that descriptive of farming article. 

Reviewer: ```suggestion
However, there are also words such as "otte" (eight), "september", and "ligesom" (like), which are not that descriptive of articles about agriculture. 
```
